src/switch_core_sqldb.c

```diff
@@ -2835,14 +2836,21 @@ static void core_event_handler(switch_event_t *event)
 		int i = 0;
 
 
+		switch_thread_rwlock_rdlock(sql_manager.RWLOCK);
 		for (i = 0; i < sql_idx; i++) {
-			if (switch_stristr("update channels", sql[i]) || switch_stristr("delete from channels", sql[i])) {
-				switch_sql_queue_manager_push(sql_manager.qm, sql[i], 1, SWITCH_FALSE);
+			if (!sql_manager.qm) {
+				switch_log_printf(SWITCH_CHANNEL_LOG, SWITCH_LOG_DEBUG1, "DROP [%s]\n", sql[i]);
```

Comment:
I'm not sure what this actually does ? Why wouldn't these be a sql_manager.qm (ps, I got no idea what this are, just that you debug print DROP yet feel that something should be handeled here )?